src/persistence.jl

```diff
@@ -1 +1,24 @@
 # Here put file load/file write functions
+function loadfast5(path)
```

Comment:
name function loadnanoporefast5, add defined types path::String, add some function description with intended input/output, add some tests for this function

Reply:
also we need to add this function to exports, then do the tests